src/hifz/utils.py

```diff
@@ -63,6 +64,10 @@ def load_progress(cls, file_path: Path) -> "CardSession":
         cards = [Card.from_dict(card_data) for card_data in session_data["cards"]]
         return cls(cards=cards, strategy=strategy)
 
+    def aggregate_statistics(self) -> dict[str, Any]:
+        """Gets global statistics from the Strategy."""
+        return self.strategy.aggregate_statistics(self.cards)
+
```

Comment:
For now, this method isn't necessary. In the engine you can simply call `self.session.strategy.aggregate_statistics(...)`. Same thing is done for the feedback mechanism.

Reply:
I'll keep this method because otherwise, in the visualizer, I'd have to write `engine.session.strategy.aggregrate_statistics(engine.session.cards)` Because the cards belong to the session. And I think its verbose that way.